VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/joh-waiting-room/joh-waiting-room.component.html

```diff
@@ -44,7 +44,7 @@
     <video
       appForcePlayVideo
       #incomingFeed
-      (playing)="updateVideoStreamMuteStatus()"
+      [mute]="shouldMuteHearing()"
```

Comment:
Just to put another option out there, instead of using the directive could we just bind this function to the muted attribute here?

The default behavior is for muted to be false anyway so even the places where we don't pass a value to the mute input should still work as expected.